[stable/yugabyte] set workingDir to /mnt/disk0

bhavin192 opened the pull request:
- If host has kernel.core_pattern set to some relative path like `core.%e.%p.%t`, then the core dumps will get saved in the volume mounted at `/mnt/disk0`

**Note**: after this change, `kubectl exec …` commands will have to use absolute paths to binaries from bin directory, i.e. `/home/yugabyte/bin/ysqlsh` instead of `bin/ysqlsh`.

### Scenarios tested
- Installation with `storage.ephemeral=true` and `storage.ephemeral=false`
- Installation with `tls.enabled=true` and `tls.enabled=false`

### How to check if workingDir is set correctly
- Output of given commands should match
  ```console
  $ kubectl exec -it yb-master-1 -nyb-chart -- bash
  [root@yb-master-1 disk0]# pwd
  /mnt/disk0
  ```

Fixes #11

ramkumarvs commented on stable/yugaware/templates/_helpers.tpl:
```diff
@@ -28,5 +28,5 @@ If release name contains chart name it will be used as a full name.
 Create chart name and version as used by the chart label.
 */}}
 {{- define "yugaware.chart" -}}
```
is this used at all, we can switch to using this template function as well

bhavin192 committed:
[stable/yugabyte] set workingDir to /mnt/disk0

- If host has kernel.core_pattern set to some relative path like
  `core.%e.%p.%t`, then the core dumps will get saved in the volume
  mounted at `/mnt/disk0`

Signed-off-by: Bhavin Gandhi <bhavin7392@gmail.com>

iSignal commented on stable/yugabyte/templates/service.yaml:
```diff
@@ -246,6 +246,7 @@ spec:
         {{ else }}
 {{ toYaml $root.Values.resource.tserver | indent 10 }}
         {{ end }}
+        workingDir: "/mnt/disk0"
```
Add a comment linking to the issue number and explaining why we did this change.
